WpfPlus/Styles/TabControlStyle.xaml

```diff
@@ -72,6 +94,31 @@
                                               ContentSource="SelectedContent" />
                         </Border>
                     </Grid>
+
```

Comment:
Please remove this and following empty lines

Reply:
Blank lines removed.